src/lib/experimental/error.ts

```diff
@@ -1,10 +1,11 @@
-// import { trap } from './ic_apis/trap'; // TODO why does this break bitcoin_psbt examples? https://github.com/demergent-labs/azle/issues/2003
 import { trap } from './ic/trap';
 
 export function handleUncaughtError(rawError: any): never {
-    const error = rawError instanceof Error ? rawError : new Error(rawError);
-
-    const azleError = `Uncaught ${error.name}: ${error.message}${error.stack}`;
-
-    trap(azleError);
+    if (rawError instanceof Error) {
```

Comment:
Can experimental use the stable trap? Could we have just one handleUncaughtError for both stable and experimental